deps/chakrashim/src/jsrtcontextshim.h

```diff
@@ -106,37 +102,32 @@ class ContextShim {
 
   IsolateShim * isolateShim;
   JsContextRef context;
-  bool initialized = false;
+  bool initialized;
   bool exposeGC;
   JsValueRef keepAliveObject;
   int builtInCount;
   JsValueRef globalObjectTemplateInstance;
 
-  JsValueRef trueRef = JS_INVALID_REFERENCE;
-  JsValueRef falseRef = JS_INVALID_REFERENCE;
-  JsValueRef undefinedRef = JS_INVALID_REFERENCE;
-  JsValueRef nullRef = JS_INVALID_REFERENCE;
-  JsValueRef zero = JS_INVALID_REFERENCE;
-  JsValueRef globalConstructor[GlobalType::_TypeCount] =
-                                { JS_INVALID_REFERENCE };
-  JsValueRef globalObject = JS_INVALID_REFERENCE;
-  JsValueRef proxyOfGlobal = JS_INVALID_REFERENCE;
-  JsValueRef reflectObject = JS_INVALID_REFERENCE;
-  JsValueRef reflectFunctions[ProxyTraps::TrapCount] =
-                               { JS_INVALID_REFERENCE };
-
-  JsValueRef globalPrototypeFunction[GlobalPrototypeFunction::_FunctionCount] =
-                                      { JS_INVALID_REFERENCE };
+  JsValueRef trueRef;
+  JsValueRef falseRef;
+  JsValueRef undefinedRef;
+  JsValueRef nullRef;
+  JsValueRef zero;
+  JsValueRef globalConstructor[GlobalType::_TypeCount];
+  JsValueRef globalObject;
+  JsValueRef proxyOfGlobal;
+
+  JsValueRef globalPrototypeFunction[GlobalPrototypeFunction::_FunctionCount];
   JsValueRef getOwnPropertyDescriptorFunction;
 
   JsValueRef promiseContinuationFunction;
   std::vector<void*> embedderData;
 
 #define DECLARE_CHAKRASHIM_FUNCTION_GETTER(F) \
  public: \
-JsValueRef Get##F##Function(); \
+   JsValueRef Get##F##Function(); \
```

Comment:
> It is intent by 2 wrt public:

All other code indent by 2 wrt to line beginning, even when "public:" indents by 1.